components/HackneyDS/Dialog/index.js

```diff
@@ -1,8 +1,9 @@
 import React, { useEffect, useState } from 'react';
 import { createPortal } from 'react-dom';
 import { CircleCloseIcon } from '../../Icons';
+import { useScrollLock } from '../../../service';
```

Comment:
does 'service' work here?

Reply:
It's storybook component, it will cause error